features/0428-prepare-issue-rich-credential/README.md

```diff
@@ -0,0 +1,83 @@
+# 0428: Prerequisites to Issue Rich Credential
+- Authors: [Brent Zundel](<brent.zundel@evernym.com>), [Ken Ebert](<ken@sovrin.org>)
+- Status: [PROPOSED](/README.md#proposed)
+- Since: 2020-02-20
+- Status Note: Part of proposed Rich Schema capabilities for credentials 
+- Supersedes: 
+- Start Date: 2020-02-19 
+- Tags: [feature](/tags.md#feature), [rich-schemas](/tags.md#rich-schemas)
+
+## Summary
+
+Describes the prerequisites an issuer must ensure are in place before issuing a
+rich credential.
+
+## Motivation
+
+To inform issuers of the steps they should take in order to make sure they have
+the necessary rich schema objects in place before they use them to issue
+credentials.
+
+## Tutorial
+
+###Rich Schema Credential Workflow
+
+1. The issuer checks the ledger to see if the credential definition he wants to
+use is already present.
+   1. If not, the issuer checks the ledger to see if the mapping he wants to use
+   is already present.
+      1. If not, the issuer checks the ledger to see if the schemas he wants to
+      use are already present.
+         1. If not, anchor the context used by each schema to the ledger.
```

Comment:
Should we clarify what we mean by anchoring here? Is it writing the full schema/mapping's content to the ledger, or just an anchor/digital_fingerprint/hash of the full content?

Reply:
I think whether the full contents of the schema/mapping should be on the ledger versus a digital hash is determined by the user's trust model, so both should be allowed. I think using the more generic term `anchor` allows for either interpretation.

Reply:
I agree as we are talking about an Aries RFC here. 
Once we write a ledger-specific RFC (Indy HIPE for example), then we may need to clarify the anchoring term in terms of supported transactions there.